.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+labels: bug
+---
+
+**Describe your environment.**
+Describe any aspect of your environment relevant to the problem, including your SDK version, platform, OS version, etc. If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on master.
```

Comment:
We can ask more specific question  - .NET Core version, .NET Framework version? (value of `TargetFramework` in your .csproj)

Reply:
Added.